tasks/main.yml

```diff
@@ -13,6 +14,16 @@
       paths:
         - "{{ role_path }}/vars"
 
+- name: Get servies list using platform specific services manager
+  include_tasks: "{{ item }}"
+  with_first_found:
+    - "services-el{{ ansible_facts['distribution_major_version'] }}.yml"
+
+- name: Check that variable 'services' is defined
```

Comment:
Ansible variables created by `register` are global so must be prefixed by the role name - so use `timesync_services`